lesson6/task2/server/server.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+var (
+	// our message
+	message = "Hello there!"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	// send Not found in such case
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+	switch r.Method {
+	case "GET":
```

Comment:
better to use constants from http package here
see https://pkg.go.dev/net/http#pkg-constants